src/lib/utils/__tests__/classNames.js

```diff
@@ -0,0 +1,34 @@
+import { classNames } from '../classNames';
+
+describe('classNames', () => {
+  it('returns filtered class names', () => {
+    const result = classNames(
+      'class-1',
+      'class-2 class-3',
+      ' ',
+      0,
+      1,
+      null,
+      undefined,
+      true,
+      false,
+    );
+
+    expect(result).toEqual('class-1 class-2 class-3');
+  });
+
+  it('returns undefined if all class names are filtered out', () => {
+    const result = classNames(
+      ' ',
+      ' ',
```

Comment:
How about `''`?

Reply:
We can but it is not necessary in my opinion. Because '' is subset of ' '. But if you want, we can add it, but it is unnecessary as we do not have fully covered tests.

Reply:
OK, didn't know that.